substrate/frame/assets/src/lib.rs

```diff
@@ -495,6 +495,10 @@ pub mod pallet {
 		Created { asset_id: T::AssetId, creator: T::AccountId, owner: T::AccountId },
 		/// Some assets were issued.
 		Issued { asset_id: T::AssetId, owner: T::AccountId, amount: T::Balance },
+		/// Some assets were deposited (e.g. for transaction fees).
+		Deposit { asset_id: T::AssetId, who: T::AccountId, amount: T::Balance },
+		/// Some assets were withdrawn from the account (e.g. for transaction fees).
+		Withdraw { asset_id: T::AssetId, who: T::AccountId, amount: T::Balance },
```

Comment:
Just to be consistent with the other events
```suggestion
		Deposited { asset_id: T::AssetId, who: T::AccountId, amount: T::Balance },
		/// Some assets were withdrawn from the account (e.g. for transaction fees).
		Withdrawn { asset_id: T::AssetId, who: T::AccountId, amount: T::Balance },
```